src/theory/quantifiers/ce_guided_pbe.cpp

```diff
@@ -61,6 +55,15 @@ void print_role( const char * c, unsigned r ){
   }
 }
 
+void CegConjecturePbe::print_strat( const char * c, unsigned s ){
```

Comment:
It would be cleaner to have an enum-to-string function instead of `print_strat`. Then you could use Trace() << strat_to_string() ... and don't have code like `Trace() << ...;
 print_strat(...);
Trace() << ...;` 

Same applies to print_role(...).

Reply:
this code may change rapidly soon, so I'm not sure yet how to print these

Reply:
Yes, but still, if you use `Trace(c) << print_strat (...)` and disable tracing, the function will not be called. As it is now, the function will always be called.

Reply:
yes i know, but this function is not called in any bottleneck areas. If it were, it can be guarded by if(Trace.isOn(c)){...}
Like I said, this will be probably be refactored soon.

Reply:
Yeah i know it's not a matter of bottlenecks, but a matter of style/cleaner code (if (Trace.isOn(c)) would not be cleaner in this case.

Reply:
indeed, my point is this strategy/role will likely soon rapidly evolve into something more than enum's.  Hence there isn't a need to clean this yet until the code is more stable.

Reply:
I agree with Mathias. Also, we should avoid to merge code into master that "needs to be refactored" from the beginning. Since if it doesn't get refactored / cleaned up later, it remains and doesn't get fixed.

Reply:
The issue is size of commits.  I would agree that we should not add new code that needs to be refactored, but in this case this is old code that needs to be refactored.